UpdateAndFixes

ppiova opened the pull request:
## What's new? ##
--Style Modification in Navigation Bar  - Closes #42  

--Bug fix in iOS when from DetailsJobView go back to ListJobs crash

--Android Primary Color Change

--Nuggets References Update  -Closes #48 

--The section "More jobs of this company" was commented on DetailsJobs => Set up a service to obtain all the jobs of a company 

--Navigator navigation to company URL added. - Closes #49 

--Navigate URL company button changed the font size for the Android platform.


### Visual aids ###
Demo Video: https://youtu.be/4eYqK-fuZLc

![01](https://user-images.githubusercontent.com/7376295/101550226-b254b580-398d-11eb-8e3d-5f4a841b46e8.png)
![02](https://user-images.githubusercontent.com/7376295/101550232-b54fa600-398d-11eb-8421-01c08dff50b6.png)
![03](https://user-images.githubusercontent.com/7376295/101550238-b7196980-398d-11eb-993f-6be8532beda2.png)
![04](https://user-images.githubusercontent.com/7376295/101550252-baacf080-398d-11eb-9b0a-01feb8aa16d7.png)
![05](https://user-images.githubusercontent.com/7376295/101550259-bc76b400-398d-11eb-8d70-e4dd26620de6.png)

ppiova committed:
--Style Modification in Navigation Bar

--Bug fix in iOS when from DetailsJobView go back to ListJobs crash

--Android Primary Color Change

--Nuggets References Update

--The section "More jobs of this company" was commented on DetailsJobs => Set up a service to obtain all the jobs of a company

--Navigator navigation to company URL added.

--Navigate URL company button changed the font size for the Android platform.

sact1909 commented on src/App/ViewModels/JobDetailViewModel.cs:
```diff
@@ -10,19 +10,50 @@
         public JobDetailViewModel()
         {
 
+            NavigateToURLCommand = new DelegateCommand(async () => await NavigateToURL());
```
Bro pudieras crear esto en un metodo void que se llame RegisterComands y luego simplemente inicializar el metodo aqui en el constructor, eso es con el fin de que no se sigan agregando instanciaciones de commands en el mismo constructor para manternerlo lo mas limpio posible, es una sugerencia, recuerda que aunque ahora esta vista tiene un solo command, le faltan algunos, asi que seria bueno que vayamos creando un standar en el codigo desde ahora... 

### Metodo 
![image](https://user-images.githubusercontent.com/23615129/101590696-ea252280-39b8-11eb-91ba-353b2ff6fcf9.png)

### Constuctor
![image](https://user-images.githubusercontent.com/23615129/101590718-f7daa800-39b8-11eb-93ff-5fffe0642993.png)

ppiova replied:
Perfecto. Lo dejaría para implementer luego  cuando se agreguen mas commands, por ejemplos share en Socials. Ahora solo está este por el momento y lo agregué cuando estaba solucionando el fix en iOS y me lleve la sorpresa que no estaba implementado esa funcionalidad.

ppiova replied:
![image](https://user-images.githubusercontent.com/7376295/101658876-35284f80-3a24-11eb-90c6-91aaf4304342.png)

sact1909 commented on src/App/ViewModels/JobDetailViewModel.cs:
```diff
@@ -10,19 +10,50 @@
+                });
+
+            }
+            catch (Exception ex)
```
Es posible mostrar un mensaje PopUp con la libreria que tu usas de mensajes, para cuando capture la excepcion, le muestre lo que esta sucediendo al cliente?  seria algo como por ejemplo, lo mismo que tienes comentado 

> "No Browser Application available to open"

ppiova replied:
Si, se podría agregar ese Alert, además cuando implemente en otro feature los crashes de AppCenter

ppiova replied:
![Alert-Navegador](https://user-images.githubusercontent.com/7376295/101658759-15912700-3a24-11eb-885a-9d04a9e00fc3.PNG)

Se utilizó Using the Page Dialog Service
https://prismlibrary.com/docs/xamarin-forms/dialogs/page-dialog-service.html

sact1909 replied:
Bien, aprobado

ppiova committed:
Added eb JobDetailsViewModel RegisterCommands ()
Added DisplayAlertAsync to be able to show message when device has disable browsers

sact1909 commented:
Aprobado
